c7n/cli.py

```diff
@@ -297,8 +295,23 @@ def setup_parser():
     # access.add_argument(
     #    '-m', '--access', default=False, action='store_true')
 
-    run_desc = ("Execute the policies in a config file")
-    run = subs.add_parser("run", description=run_desc, help=run_desc)
+    run_desc = "\n".join((
+        "Execute the policies in a config file",
+        "",
+        "Multiple regions can be passed in, as can the symbolic region 'all'. ",
+        "",
+        "When running across multiple regions, policies targeting resources in ",
+        "regions where they do not exist will not be run. The output directory ",
```

Comment:
Running against a single region has the same behavior (it will filter it out if the region is not valid for the policy).  

But I guess what you're getting at is that people can run a set of policies for different resources and specify any regions and only the appropriate ones will be run.